docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -474,6 +474,16 @@ <h1>Samza Configuration Reference</h1>
                         These coordination utils are currently used for intermediate stream creation.
                     </td>
                 </tr>
+
+                <tr>
+                    <td class="property" id="job.logged.store.base.dir">job.logged.store.base.dir</td>
```

Comment:
Should clarify why this is *logged.*store.base.dir, since non-changelogged stores are stored in YARN container directory.

Reply:
modified the configuration name